### PR TITLE
Remove Deprecations from React Native dependencies

### PR DESCRIPTION
**Pull request by Wunderkemmer**
Updated ViewPager

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,26 +1,26 @@
 const React = require('react');
-const { Component } = React;
-const { ViewPropTypes } = ReactNative = require('react-native');
+const { ViewPropTypes, } = ReactNative = require('react-native');
```
Comment (lunarraid) [inline index.js:2]:
```suggestion
const { ViewPropTypes } = ReactNative = require('react-native');
```

File: package.json [hunk 2]
```diff
@@ -29,6 +29,7 @@
   },
   "homepage": "https://github.com/brentvatne/react-native-scrollable-tab-view#readme",
   "dependencies": {
+    "@react-native-community/viewpager": "^2.0.2",
```
Comment (lunarraid) [inline package.json:32]: Don't forget to also bump the version for this module above.

**Review by lunarraid — COMMENTED** [review]
Just a misplaced comma and version bump. If for some reason using the original module instead of our fork isn't compatible, this is fine.

File: index.js (hunk 1 above)
Comment (Wunderkemmer) [reply index.js:2]: not according to the style of that file/project. it's consistently terminating commas :/

**Comment by Wunderkemmer** [comment]
Merging!

**Pull request merged**